### PR TITLE
DOCS-11582 user teams

### PR DESCRIPTION
**Pull request by wwarner-salesforce**
DOCS-11582 user teams

### CODE REVIEW

File: modules/ROOT/pages/guided-setup.adoc [hunk 1]
```diff
@@ -66,20 +66,24 @@ Otherwise, provide these app details:
...
+Give community users the Exchange viewer permission:
 
 . Ensure that you have Exchange Administrator permissions.
-. In your Anypoint Platform organization, navigate to *Access Management* and click *Roles*.
-. In *Exchange Viewers*, set a new external group called `Community User` and save it.
+. In your Anypoint Platform organization, navigate to *Access Management* and click *Teams*.
+. Create a new team named *Community Users*.
+. Click *Settings*.
+. Select the parent team.
+. In *External IdP Groups*, set *Group Name* to *Community Users* and set *Type* to *Member*.
```
Comment (ferferrarazzo) [inline modules/ROOT/pages/guided-setup.adoc]:
```suggestion
. In *External IdP Groups*, set *Group Name* to *Community User* and set *Type* to *Member*.
```

File: modules/ROOT/pages/manual-setup.adoc [hunk 2]
```diff
@@ -113,17 +113,21 @@ Replace `ORGANIZATION_ID` with the organization ID from your Anypoint account.
...
+Give community users the Exchange viewer permission:
 
 . Ensure that you have Exchange Administrator permissions.
-. In your Anypoint Platform organization, navigate to *Access Management* and click *Roles*.
-. In *Exchange Viewers*, set a new external group called `Community User` and save it.
+. In your Anypoint Platform organization, navigate to *Access Management* and click *Teams*.
+. Create a new team named *Community Users*.
+. Click *Settings*.
+. Select the parent team.
+. In *External IdP Groups*, set *Group Name* to *Community Users* and set *Type* to *Member*.
```
Comment (ferferrarazzo) [inline modules/ROOT/pages/manual-setup.adoc]:
```suggestion
. In *External IdP Groups*, set *Group Name* to *Community User* and set *Type* to *Member*.
```

**Review by ferferrarazzo — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/guided-setup.adoc (hunk 1 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/guided-setup.adoc]: Done. Thank you!

File: modules/ROOT/pages/manual-setup.adoc (hunk 2 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/manual-setup.adoc]: Done. Thank you!

**Review by ferferrarazzo — DISMISSED** [review]

File: modules/ROOT/pages/guided-setup.adoc [hunk 3]
```diff
@@ -66,20 +66,24 @@ Otherwise, provide these app details:
 
 == Configure Users
 
-Give your user the *Exchange Contributor* role:
+Give your user the Exchange contributor permission:
```
Comment (kbostrom) [inline modules/ROOT/pages/guided-setup.adoc:69]: Nice! I like seeing names replaced by functions in narrative.

File: modules/ROOT/pages/guided-setup.adoc [hunk 4]
```diff
@@ -66,20 +66,24 @@ Otherwise, provide these app details:
...
-Map community user to Anypoint role:
+Give community users the Exchange viewer permission:
 
 . Ensure that you have Exchange Administrator permissions.
-. In your Anypoint Platform organization, navigate to *Access Management* and click *Roles*.
-. In *Exchange Viewers*, set a new external group called `Community User` and save it.
+. In your Anypoint Platform organization, navigate to *Access Management* and click *Teams*.
+. Create a new team named *Community Users*.
+. Click *Settings*.
+. Select the parent team.
```
Comment (kbostrom) [inline modules/ROOT/pages/guided-setup.adoc]: Maybe we should be explicit here about which team is the parent?
Is it your new team? Some existing team? or just "the larger team to which you are adding your new team" (or something similar)?

File: modules/ROOT/pages/guided-setup.adoc [hunk 5]
```diff
@@ -66,20 +66,24 @@ Otherwise, provide these app details:
...
+. In *External IdP Groups*, set *Group Name* to *Community User* and set *Type* to *Member*.
+. Click *Permissions* > *Add permissions* and add the permission *Exchange Viewer*.
 +
-If you work with different business groups, you must create a new external group for each business group in your organization.
+When prompted, select all business groups with any assets that community users will view.
```
Comment (kbostrom) [inline modules/ROOT/pages/guided-setup.adoc]: "will view" or "that you want community users to be able to view"?

File: modules/ROOT/pages/manual-setup.adoc [hunk 6]
```diff
@@ -113,17 +113,21 @@ Replace `ORGANIZATION_ID` with the organization ID from your Anypoint account.
 
 == User Configuration
```
Comment (kbostrom) [inline modules/ROOT/pages/manual-setup.adoc:114]: In this section, ditto edits to the same text above ^

**Review by kbostrom — DISMISSED** [review]
CE, CU

File: modules/ROOT/pages/guided-setup.adoc (hunk 3 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/guided-setup.adoc:69]: Great!

File: modules/ROOT/pages/guided-setup.adoc (hunk 4 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/guided-setup.adoc]: True! Fixed.

File: modules/ROOT/pages/guided-setup.adoc (hunk 5 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/guided-setup.adoc]: "that you want community users to be able to view". Fixed.

File: modules/ROOT/pages/manual-setup.adoc (hunk 6 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/manual-setup.adoc:114]: Done.

**Review by kbostrom — APPROVED** [review]
RA

**Pull request merged**